src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -188,14 +176,15 @@ public void writeToNBT(NBTTagCompound compound)
         compound.setString(TAG_NAME, name);
         BlockPosUtil.writeToNBT(compound, TAG_CENTER, center);
 
+        compound.setBoolean(TAG_MANUAL_HIRING,manualHiring);
         compound.setInteger(TAG_MAX_CITIZENS, maxCitizens);
 
         // Permissions
         permissions.savePermissions(compound);
 
         //  Buildings
         NBTTagList buildingTagList = new NBTTagList();
-        for (Building b : buildings.values())
+        for (AbstractBuilding b : buildings.values())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'b' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)